sites/fast-creator/app/preview/preview.tsx

```diff
@@ -18,12 +18,14 @@ import {
     fastComponentDefinitions,
     nativeElementDefinitions,
 } from "@microsoft/site-utilities";
-import { Direction } from "@microsoft/fast-web-utilities";
+import { classNames, Direction } from "@microsoft/fast-web-utilities";
 import {
     mapFASTComponentsDesignSystem,
     setupFASTComponentDesignSystem,
 } from "../configs/library.fast.design-system.mapping";
 import { elementLibraries } from "../configs";
+import { creatorOriginatorId } from "../creator";
+import { DisplayMode } from "../creator.props";
```

Comment:
We should place `DisplayMode` and `creatorOriginatorId` in the `app/utilities/shared.ts` because if we import from `creator.tsx` we will end up bulking up the distributed preview application when webpack bundles the scripts.

Reply:
Done